packages/core/src/writer.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ServerResponse as Response} from 'http';
+
+// tslint:disable:no-any;
+export type OperationRetval = any;
```

Comment:
Please don't duplicate this type definition. Keep it in single place and import it everywhere else where needed/used. Similarly to how `OperationArgs` are handled (for an example).